src/DirectionProvider.jsx

```diff
@@ -45,10 +44,10 @@ export default class DirectionProvider extends React.Component {
   }
 
   render() {
-    const { children, direction, inline } = this.props;
+    const { children, direction, inline, ...rest } = this.props;
```

Comment:
instead of `...rest`, at the least, could this be an actual `tagProps` prop or something?